src/api/actions/uploadMedia.ts

```diff
@@ -17,6 +17,6 @@ export const uploadMedia = async (
     throw Error(`Failed to upload metadata: ${e}`);
   }
 
-  const url = ipfsHashToUrl(response.ipfsHash);
+  const url = response.url;
```

Comment:
`uploadMedia` and `uploadObject` should both do `ipfsHashToUrl(response.hash);`
We want to adopt an IPFS-first approach to file uploads.